caddy/setup/rewrite_test.go

```diff
@@ -154,6 +159,22 @@ func TestRewriteParse(t *testing.T) {
 			&rewrite.ComplexRule{},
 		}},
 		{`rewrite {
+			status 200
+		 }`, false, []rewrite.Rule{
+			&rewrite.ComplexRule{Base: "/", Regexp: regexp.MustCompile(".*"), Status: 200},
+		}},
+		{`rewrite {
+			to /to
+			status 200
+		 }`, false, []rewrite.Rule{
+			&rewrite.ComplexRule{Base: "/", To: "/to", Regexp: regexp.MustCompile(".*"), Status: 200},
+		}},
```

Comment:
Call me pedantic, but I think these tests don't need the regexp, right? If not, maybe we could remove it to keep the case more obvious.